api/nodecontrol/start-workload-request.json

```diff
@@ -91,6 +96,21 @@
     "workload_type"
   ],
   "definitions": {
+    "encrypted_environment": {
```

Comment:
in other places this looks like `enc_environment`  and not `encrypted_environment`. Am I missing something that makes this still work?

Reply:
i named it enc_environment in the json gen and encrypted_environment in the proto3 gen :cone-of-shame: